components/prism-rescript.js

```diff
@@ -0,0 +1,44 @@
+var operator = /\.{3}|:[:=]|\|>|->|=(?:==?|>)?|<=?|>=?|[|^?'#!~`]|[+\-*\/]\.?|\b(?:asr|land|lor|lsl|lsr|lxor|mod)\b/;
```

Comment:
This is a top-level variables that will leak into global scope. Please wrap the whole file into an IIFE. [Example](https://github.com/PrismJS/prism/blob/master/components/prism-java.js#L1).

Reply:
Done